src/prefer-arrow-functions.spec.ts

```diff
@@ -492,14 +495,6 @@ const invalidAndHasBlockStatement = [
     code: 'async function foo(a) { console.log(3); }',
     output: 'const foo = async (a) => { console.log(3); };',
   },
-  {
```

Comment:
The new `RuleTester` by ESLint fails if you have duplicated tests, so I had to remove a bunch of duplicated tests.